lib/core/src/sdk.rs

```diff
@@ -3479,9 +3489,8 @@ impl LiquidSdk {
                             bip353_address: req.bip353_address,
                         }
                     } else {
-                        return Err(LnUrlPayError::Generic {
-                            err: "SendDestination for LNURL Pay is not BOLT11 invoice".to_string(),
-                        });
+                        // Destination is a Liquid address for the drain request
+                        prepare_response.destination
```

Comment:
Wouldn't it be more robust to use pattern matching on `prepare_response.destination`?

```
let destination = match prepare_response.destination {
    SendDestination::Bolt11 { invoice, .. } => ...,
    SendDestination::LiquidAddress { .. } => ...,
    ...
```

Reply:
Does this also imply we should add the optional `bip353_address` to `SendDestination::LiquidAddress` for LNURL-pay drain's via BIP 353?

Reply:
Good catch, added [c4f1559](https://github.com/breez/breez-sdk-liquid/pull/776/commits/c4f15590e8a2d7b5f12551155359c63883d5a0e2)